CHANGELOG.md

```diff
@@ -11,6 +11,50 @@ This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the chang
 
 <!-- towncrier release notes start -->
 
+## [1.3.0](https://github.com/opsmill/infrahub-sdk-python/tree/v1.3.0) - 2024-12-30
+
+### Added
+
+#### Testing library (**Alpha**)
+
+A new collection of tools and utilities to help with testing is available under `infrahub_sdk.testing`.
+
+The first component available is a `TestInfrahubDockerClient`, a pytest Class designed to help creating integration tests based on Infrahub. See a simple example below to help you get started.
+
+> the installation of `infrahub-testcontainers` is required
+
+```python
+import pytest
+
+from infrahub_sdk import InfrahubClient
+from infrahub_sdk.testing.docker import TestInfrahubDockerClient
+
+class TestInfrahubNode(TestInfrahubDockerClient):
+
+    @pytest.fixture(scope="class")
+    def infrahub_version(self) -> str:
+        """Required (for now) to define the version of infrahub to use."""
+        return "1.0.10"
+
+    @pytest.fixture(scope="class")
+    async def test_create_tag(self, default_branch: str, client: InfrahubClient) -> None:
+        obj = await client.create(kind="BuiltinTag", name="Blue")
+        await obj.save()
+        assert obj.id
+```
+
+### Changed
+
+- The Pydantic models for the schema have been split into multiple versions to align better with the different phase of the lifecycle of the schema.
```

Comment:
Should we highlight this a bit more to indicate that if people were importing the repository classes from `infrahub_sdk.schema` they would need to change those imports? While I don't see why end users would have to import these classes you never know.

Reply:
I'm planning to send a quick note in Discord as well ... as you mentioned I don't think a lot of users are using these models today
also, the import should still work ... the main difference is that some attributes have been removed from the models